Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/ProjectStatsController.java

```diff
@@ -34,92 +54,162 @@
+
+
+
+	private int getCount(List<IConceptStats> conceptList) {
+		int cnt = Integer.parseInt(env.getProperty("project.stats.topcount"));
+		int len = conceptList.size() > cnt ? cnt : conceptList.size();
+		return len;
+	}
+
+	private JSONArray getTopConceptsJson(List<IConceptStats> conceptsList,
+			int length) throws JSONException {
+		JSONArray jsonArray = new JSONArray();
+
+		for (int i = 0; i < length; i++) {
+			JSONObject jsonObject = new JSONObject();
+			IConceptStats conceptStats = conceptsList.get(i);
+			jsonObject.put("conceptId", conceptStats.getConceptId().replace("\"", ""));
+			jsonObject.put("description", conceptStats.getDescription().replace("\"", ""));
+			jsonObject.put("label", conceptStats.getLemma().replace("\"", ""));
+			jsonObject.put("count", conceptStats.getCount());
+			jsonArray.put(jsonObject);
+		}
+		return jsonArray;
+	}
+
+	/**
+	 * This method returns Json Array containing date and corresponding count of number 
+	 * of networks that were Approved/Rejected/Submitted depending on 'status' 
+	 * @author Bharath Srikantan
+	 * @throws JSONException 
+	 *
+	 */
+	private JSONArray getContributionCountByStatus (List<INetwork> networks, String status) throws JSONException
+	{
+		String DATE_FORMAT = "dd-MMM-yy";
+		SimpleDateFormat sdf = new SimpleDateFormat(DATE_FORMAT);
+		JSONArray contributionsJson = new JSONArray();
+		HashMap<String, Integer> contributionCount = new HashMap<String, Integer>();
+
+		for(INetwork network : networks)
+		{
+			if(network.getStatus().equals(status) || status.equals("SUBMITTED"))
+			{
+				if(contributionCount.containsKey(sdf.format(network.getCreatedDate())))
+				{
+					contributionCount.put(sdf.format(network.getCreatedDate()), contributionCount.get(sdf.format(network.getCreatedDate()))+1);
+				}
+				else
+				{
+					contributionCount.put(sdf.format(network.getCreatedDate()), 1);
+				}
+			}
+		}
+		for(Entry<String, Integer> entry : contributionCount.entrySet())
+		{
+			JSONObject jsonObject = new JSONObject();
+			jsonObject.put("date", entry.getKey());
+			jsonObject.put("count", entry.getValue());
+			contributionsJson.put(jsonObject);
+		}
+		return contributionsJson;    	
+	}
+
+	/**
+	 * This method gives the visualization of how often concepts appear in the
+	 * networks
+	 * 
+	 * @author Bharath Srikantan & Ajay Modi
+	 * @param projectUnixName
+	 *            The project unix name
+	 * @param model
+	 *            Model
+	 * @return view
+	 * @throws JAXBException
+	 * @throws QuadrigaStorageException
+	 * @throws JSONException 
+	 */
+
+	@RequestMapping(value = "sites/{projectUnixName}/statistics", method = RequestMethod.GET)
+	public String showProjectStatistics(
+			@PathVariable("projectUnixName") String projectUnixName, Model model, Principal principal)
+					throws JAXBException, QuadrigaStorageException, JSONException {
+
+		IProject project = projectManager.getProjectDetailsByUnixName(projectUnixName);

```

Comment:
Take a look at this and try using the new annotation (you might have to merge devleop):
https://diging.atlassian.net/wiki/display/QUAD/Injecting+projects+by+unix+names